app/models/permit_application.rb

```diff
@@ -209,10 +209,12 @@ def take_form_customizations_snapshot_if_submitted
 
   def notify_user_application_updated
     return if new_record?
-
-    return unless saved_change_to_viewed_at? || saved_change_to_reference_number?
-
-    PermitHubMailer.notify_application_updated(submitter, self).deliver_later
+    viewed_at_change = previous_changes.dig("viewed_at")
```

Comment:
Is it possible to use this api `attribute_changed?(attr_name)` , I believe it should give a boolean for if the attribute was changed

Reply:
if not good to go

Reply:
this is the only way I could get it to work. I'm trying to find not when it was changed but specifically when it was changed from nil